geode-core/src/main/java/org/apache/geode/management/internal/ManagementResourceRepo.java

```diff
@@ -132,7 +132,7 @@ public boolean keyExistsInLocalMonitoringRegion(String key) {
    */
   public Object getEntryFromLocalMonitoringRegion(ObjectName name) {
 
-    return localMonitoringRegion.get(name);
+    return localMonitoringRegion.get(name.toString());
```

Comment:
This one is scary. Your change looks correct, but I would expect MBean monitoring to be broken. Maybe we don't have any test coverage.